documentation/commands/git-ship.md

```diff
@@ -20,6 +20,7 @@ into the main branch, leading to linear history on the main branch.
 * pulls remote updates for `<branch_name>`
```

Comment:
I can't comment on line 17 in the PR as you didn't touch it, but do you think this reads better?

> Squash-merges the current branch, or `<branch_name>` if given, into the main branch, resulting in linear history on the main branch.